packages/open-api-gateway/src/project/open-api-gateway-ts-project.ts

```diff
@@ -57,22 +73,42 @@ export class OpenApiGatewayTsProject extends TypeScriptProject {
    */
   public readonly generatedTypescriptClient: TypeScriptProject;
 
+  /**
+   * The directory in which the OpenAPI spec file(s) reside, relative to the project outdir
+   */
+  public readonly specDir: string;
```

Comment:
This should be optional, otherwise the user will get type errors if not providing a value. Also please include@default "spec" to jsdocs.

Reply:
Good catch :)

Reply:
Same as above, this is a property rather than an input :)